src/test/java/com/wikia/webdriver/testcases/auth/ForcedLoginTests.java

```diff
@@ -63,15 +58,11 @@ public void anonCanLogInViaUserLoginPage() {
     WikiBasePageObject base = new WikiBasePageObject();
     base.openSpecialUpload(wikiURL);
     base.verifyLoginRequiredMessage();
-    SpecialUserLoginPageObject special = base.clickLoginOnSpecialPage();
-    new SignInPage(driver)
-        .getLoginArea()
-        .typeUsername(credentials.userName10)
-        .typePassword(credentials.password10)
-        .clickSignInButtonToSignIn()
-        .verifyUserLoggedIn(credentials.userName10);
-
-    Assertion.assertTrue(special.isStringInURL(URLsContent.SPECIAL_UPLOAD));
+    base.clickLoginOnSpecialPage();
+    new AttachedSignInPage().login(credentials.userName10, credentials.password10);
```

Comment:
I simply didn't change the old implementation, will make it more clear in IRIS-3963